src/Types/Users.hs

```diff
@@ -31,6 +32,7 @@ module Types.Users
   )
 where
 
+import           Data.Foldable            (find)
```

Comment:
A tiny, tiny nit: but the whitespace convention on this import doesn't match the adjacent ones. I'll fix it if you don't get to it. (Granted, we don't do this consistently across the codebase, largely because different team members have different preferences / muscle memory.)

Reply:
Oh sorry I missed that. Pushed a fix for that.